uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/passengerdetails/PassengerDetailsView.kt

```diff
@@ -103,9 +106,63 @@ class PassengerDetailsView @JvmOverloads constructor(
         lastNameInput.onFocusChangeListener = createFocusChangeListener()
         emailInput.onFocusChangeListener = createFocusChangeListener()
         mobileNumberInput.onFocusChangeListener = createFocusChangeListener()
+
+        firstNameLayout.hint = firstNameLayout.hint.toString() + "*"
+        lastNameLayout.hint = lastNameLayout.hint.toString() + "*"
+        emailLayout.hint = emailLayout.hint.toString() + "*"
+        mobileNumberLayout.hint = mobileNumberLayout.hint.toString() + "*"
+
+        firstNameLayout.editText?.accessibilityDelegate = object : View.AccessibilityDelegate() {
+            override fun onInitializeAccessibilityNodeInfo(
+                host: View,
+                info: AccessibilityNodeInfo
+            ) {
+                super.onInitializeAccessibilityNodeInfo(host, info)
+                info.text = firstNameLayout.hint.toString().removeSuffix("*") +  " " + retrieveLocalizedText(R.string.kh_uisdk_generic_mandatory_field)
```

Comment:
This part of getting the right text seems repetitive enough to justify a private method that takes an initial string as input and provides `text.removeSuffix("*") +  " " + retrieveLocalizedText(R.string.kh_uisdk_generic_mandatory_field)` as output. Especially if we ever decide to use a different pattern for these fields.

Reply:
Also, instead of using `firstNameLayout.hint.toString().removeSuffix("*")` we could go for `kh_uisdk_placeholder_first_name` which has 2 benefits:
- doesn't need tinkering with, there's no asterisk 
- provides unity with iOS implementation so in case we change the value it will be reflected in both platforms. 
Same goes for the other fields as we may want to change "Last Name" to "Surname" for example, or "Mobile Phone" to "Mobile Number", etc.

Reply:
Sorry, I added the wrong key as an example in my second comment. Double checking the key, I realise that on iOS we're using `kh_uisdk_first_name_hint` for the title too. So all good. Disregard my second comment